src/dispatch/incident/flows.py

```diff
@@ -140,68 +140,74 @@ def inactivate_incident_participants(incident: Incident, db_session: Session):
     )
 
 
-@background_task
-def incident_create_flow(*, organization_slug: str, incident_id: int, db_session=None) -> Incident:
-    """Creates all resources required for new incidents."""
-    # we get the incident
-    incident = incident_service.get(db_session=db_session, incident_id=incident_id)
-
+def incident_create_resources(*, incident: Incident, db_session=None) -> Incident:
+    """Creates all resources required for incidents."""
     # we create the incident ticket
-    ticket_flows.create_incident_ticket(incident=incident, db_session=db_session)
+    if not incident.ticket:
+        ticket_flows.create_incident_ticket(incident=incident, db_session=db_session)
 
     # we resolve individual and team participants
     individual_participants, team_participants = get_incident_participants(incident, db_session)
+    tactical_participant_emails = [i.email for i, _ in individual_participants]
```

Comment:
nit: I'd move this line after the next `if` like you have it for the incident notifications group below

Reply:
I kept `tactical_participant_emails` here because the variable is used again for storage and conference creation.